src/App.scss

```
@@ -1 +1,39 @@
-// style go here
+body {
+  margin: 0;
+  padding: 60px;
+  display: flex;
+  background-color: #dadada;
+}
+
+.article {
+  padding: 20px;
+  max-width: 1024px;
+  min-width: 600px;
+  font-weight: 400;
+  font-family: CustomSansSerif, "Lucida Grande", Arial, sans-serif;
+
+  background-color: #fff;
+  box-shadow: 0 15px 10px #363333;
+
+  &__date {
+    font-size: 15px;
+    line-height: 28px;
+    color: #79828b;
+    margin: 21px 21px 12px;
+  }
+
+  &__text {
+    margin: 0 21px 12px;
+    font-size: 17px;
+    line-height: 1.58;
```

It'd be better to specify what units are you using for your styles
```suggestion
    line-height: 1.58em;
```

---

Also, `1.58` value is a bit weird, try not to use that big precision if it's unnecessary